deepdives/build-process/README.md

```diff
@@ -0,0 +1,66 @@
+
+##### Babel
+![Babel](https://avatars0.githubusercontent.com/u/9637642?v=3&s=400)
+
+Babel can convert basically any javascript (including jsx!) you can imagine into es5. It does this by requireing users to specify a ```preset``` or set of presets that babel will use to do the compilation. You can also specify ```plugins``` to extend the features of babel. The way that you specify these is different based on the rest of the tools we use. You can look at the examples above to see how it is done in different setups.
+
+###### Readings
+- [Docs](https://babeljs.io/)
+- [What is Babel the Javascript Compiler](http://www.programwitherik.com/what-is-babel-the-javascript-compiler/)
+
+### Bundlers
+You may have noticed that as the clientside portion or your apps get bigger dealling with importing scripts, css, and images into your pages becomes a huge pain in the @#$. Wouldn't it be nice if we could require files like we can on our server? Well do I have good news for you, you have two good options, browserify and webpack.
+
+##### Browserify
+![Browserify](http://derickbailey.com/wp-content/uploads/2014/06/NewImage.png)
+
+Browserify aims to recreate the node module system on the client. This is great, it lets you use most node modules on the client as well as bundling your javascript into one (or a bunch of smaller if you prefer) concise file to serve. Since broswerify recreates the node require system the syntax is very familiar, but it falters when it comes to dealing with files that aren't common on server side programming such as css and images. You can shoe horn support for other files but it can get pretty messy.
+
+###### Readings
+- [Docs](http://browserify.org/)
+- [Browserify Handbook](https://github.com/substack/browserify-handbook)
+- [Getting Started with Browserify](http://www.sitepoint.com/getting-started-browserify/)
+
+##### Webpack
+![Webpack](https://camo.githubusercontent.com/f1d103872f836f33dbff7a74ed819004f792a4ad/687474703a2f2f7765627061636b2e6769746875622e696f2f6173736574732f6c6f676f2e706e67)
+
+Webpack is the current darling of the frontend programming world. It is an incredibly powerful bundling tool, but initally groking (understanding) it can be difficult. Weback can be configured to deal with any file type by using a variety of ```loaders``` which teach webpack how to handle each type of file. These loaders can be very powerful such as the babel loader which will compile your javascript files before bundling them. It can use any module system including both node like syntax or the es6 module system. Webpack is incredibly versatile and powerful and can also do most of the things you would need a task runner (see below) to do. It if frequently used entirely on its own.
```

Comment:
starting at Weback can be configured to... -> _rephrasing, spelling_
Webpack can be configured to deal with any file type using a variety of `loaders`. These loaders can be very powerful -- for example, the Babel loader compiles/transpiles your JavaScript files before bundling them. You can use any module system with Webpack, including the node-like `require` system or the ES6 module syntax.
(ending at "Webpack is incredibly versatile...")